src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/DSA/DSATestData.cs

```diff
@@ -67,5 +67,39 @@ internal static DSAParameters GetDSA2048Params()
                 "081025751572").HexToByteArray();
             return p;
         }
+
+        // The parameters and signature come from FIPS 186-2 APPENDIX 5. EXAMPLE OF THE DSA
+        internal static DSAParameters GetDSA1024_186_2_Params()
+        {
+            return new DSAParameters()
+            {
+                P = (
+                "8df2a494492276aa3d25759bb06869cbeac0d83afb8d0cf7cbb8324f0d7882e5d0762fc5b7210eafc2e9adac32ab7aac" +
+                "49693dfbf83724c2ec0736ee31c80291").HexToByteArray(),
+                Q = ("c773218c737ec8ee993b4f2ded30f48edace915f").HexToByteArray(),
+                G = (
+                "626d027839ea0a13413163a55b4cb500299d5522956cefcb3bff10f399ce2c2e71cb9de5fa24babf58e5b79521925c9c" +
+                "c42e9f6f464b088cc572af53e6d78802").HexToByteArray(),
+                X = ("2070b3223dba372fde1c0ffc7b2e3b498b260614").HexToByteArray(),
+                Y = (
+                "19131871d75b1612a819f29d78d1b0d7346f7aa77bb62a859bfd6c5675da9d212d3a36ef1672ef660b8c7c255cc0ec74" +
+                "858fba33f44c06699630a76b030ee333").HexToByteArray()
+            };
+        }
+
+        internal static byte[] GetDSA1024_186_2_Signature()
+        {
+            return (
+                // r
+                "8bac1ab66410435cb7181f95b16ab97c92b341c0" +
+                // s
+                "41e2345f1f56df2458f426d155b4ba2db6dcd8c8"
+                ).HexToByteArray();
+        }
+
+        internal static byte[] GetDSA1024_186_2_Data()
+        {
+            return Encoding.ASCII.GetBytes("abc");
+        }
```

Comment:
The original intent was that the three parts could be used independently (i.e. not always all 3 would be used in a given test), but since that is not the case I can combine